pegasus/router.rb

```diff
@@ -256,7 +257,14 @@ def self.load_configs_in(dir)
 
   get '/style.css' do
     content_type :css
-    Dir.glob(pegasus_dir('sites.v3',request.site,'/styles/*.css')).sort.map{|i| IO.read(i)}.join("\n\n")
+    css_last_modified = Time.at(0)
```

Comment:
Is there a reason why we prefer a 60 minute cache lifetime for this resource vs. fingerprinting + infinite cache lifetime as we do in Dashboard?  

For my own education, how often does this bundling code end up getting run?  I assume that the Varnish cache means we only have to rerun it when it expires from the cache?